docs: add more description about installing nvidia driver to specific node (#951)
harvester/docs: docs/advanced/addons/nvidiadrivertoolkit.md

### Customizing Node Affinity

You can customize the node affinity settings to meet your specific requirements. 

> For example, if you label your node with `gpu.model=A100` and `gpu.model=A40`, then you can use the following node affinity like the below. The driver is installed on nodes with specific GPU models.

Reviewer: ```suggestion
For example, if you add the labels `gpu.model=A100` and `gpu.model=A40` to nodes that use these GPU models, you can use the following node affinity settings to target the driver installation.
```
